src/renderer/mod.rs

```diff
@@ -52,6 +55,7 @@ pub struct RendererSettings {
     debug_renderer: bool,
     profiler: bool,
     underline_stroke_scale: f32,
+    group_zindex_step: u64,
```

Comment:
This new setting needs documentation added here: https://github.com/neovide/neovide/blob/main/website/docs/configuration.md#underline-automatic-scaling

Reply:
Add docs!